### DISCUSSION TITLE
Trino on Kubernetes

### DISCUSSION THREAD

**Discussion by alexortner** [trinodb/trino#6741] [Q&A] [answered]
Is there any documentation on how to run and scale Trino on Kubernetes? I can't find anything valuable via google. The old helm charts are deprecated and the only documentations is part of the enterprise version in starburstdata. I am getting somehow confused what is now the leading version of the open source Presto/Trino Version.
Any help would be very appreciated.
Thanks Alex

**Comment by bitsondatadev** [comment]
Hey Alex,

I'm not a huge kub wiz but I searched over our kubernetes channel [on slack](https://trino.io/slack.html) and someone posted an OS operator approach that you may find useful.

https://github.com/falarica/steerd-presto-operator

Word on the street is there's not much yet regarding helm charts. I'm tagging @mosabua to see if he has any updates there.

**Comment by joshthoward** [comment, marked as answer]
@alexortner 

We have new [Helm charts](https://github.com/trinodb/charts). The are not yet available on ArtifactHub (in progress), but they have been tested and have a single configuration for # workers. If you are interested in autoscaling let us know and we can add it to the backlog.
